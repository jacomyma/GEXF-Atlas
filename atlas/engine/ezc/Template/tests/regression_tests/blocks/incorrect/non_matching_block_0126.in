{* file: non_matching_block_0126.in *}
{var $foo = 1, $_false = false, $array = array( 1, 2 )}
{foreach $array as $bar}
    {elseif $foo}
        {$foo}
    {/switch}
{/foreach}

