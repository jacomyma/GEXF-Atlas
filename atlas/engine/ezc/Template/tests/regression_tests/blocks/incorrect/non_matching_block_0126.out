mock:4:12: The block {elseif} cannot be a sub-block of {foreach}. {elseif} can only be a child of an {if} block.

    {elseif $foo}
           ^
